tests/backup/backup_helper.go

```diff
@@ -1173,8 +1181,9 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration int, retr
 		return "", false, nil
 	}
 
-	task.DoRetryWithTimeout(backupSuccessCheck, time.Duration(retryDuration)*time.Minute, time.Duration(retryInterval)*time.Second)
-	bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+	_, err := task.DoRetryWithTimeout(backupSuccessCheck, time.Duration(retryDuration)*time.Minute, time.Duration(retryInterval)*time.Second)
+	log.FailOnError(err, "Verifying status for backup-%s", backupName)
```

Comment:
I thought we are just returning the error here instead of handling it inside the function

Reply:
done